pkg/runtime/runtime.go

```diff
@@ -1331,9 +1340,15 @@ func (a *DaprRuntime) getGRPCAPI() grpc.API {
 		a.getComponents,
 		a.ShutdownWithWait,
 		a.getComponentsCapabilitesMap,
+		a.extendedMetadata,
 	)
 }
 
+func (a *DaprRuntime) storeMetadata() {
```

Comment:
nitpicking: this method has a generic "storeMetadata" name when in reality it initializing DaprRuntimeVersion within the the extended metadata of the Metedata API -- this is too "meta" and abstract ;)  I would make the name a bit more more descriptive like "initializeExtendedMetadata".